vagrant/provisioning/roles/solr/templates/solr.service

```diff
@@ -1,17 +1,12 @@
 [Unit]
-Description=Solr service
+Description=SOLR Service
+
 
 [Service]
 Type=forking
-RemainAfterExit=true
-ExecStart={{ root_folder }}/app/solr/bin/init.d/solr start
-ExecStop={{ root_folder }}/app/solr/bin/init.d/solr stop
-# Starting from CentOS 7.7 we can't specify the PIDfile for 
-# solr anymore; but systemd can find it from the process 
-# table anyway.
-# PIDFile={{ root_folder }}/app/solr/bin/solr-8983.pid
 User=root
 Group=root
+PIDFile=/opt/app/arkcase/app/solr/bin/solr-8983.pid
```

Comment:
I guess we shouldn't use a hardcoded path here, but `{{ root_folder }}` instead?

Reply:
definitely need {{ root_folder }} here.